www/js/metrics-factory.js

```diff
@@ -5,37 +5,61 @@ angular.module('emission.main.metrics.factory', ['emission.plugin.kvstore'])
 .factory('FootprintHelper', function() {
   var fh = {};
   var footprint = {
-    train: 92/1609,
-    car: 287/1609,
-    ON_FOOT: 0,
-    BICYCLING: 0
+    ON_FOOT:      0,
+    BICYCLING:    0,
+    CAR:        267/1609,
+    BUS:        278/1609,
+    TRAIN:       92/1609,
+    AIR_OR_HSR: 217/1609
   }
   var readable = function(v) {
     return v > 9999? Math.round(v / 1000) + 'k kg CO₂' : Math.round(v) + ' kg CO₂';
   }
   var mtokm = function(v) {
     return v / 1000;
   }
-  fh.getFootprintRaw = function(distance, mode) {
-    if (mode === "IN_VEHICLE") {
-      return [footprint.train * mtokm(distance), footprint.car * mtokm(distance)];
-    } else {
-      return footprint[mode] * mtokm(distance);
+
+  fh.readableFormat = function(v) {
```

Comment:
Is this a replacement for `readable` that is exposed externally? If so, you can remove `readable`. I don't see any difference in functionality.

Reply:
By the way shouldn't the comparison be ```v > 999``` (or even ```v >= 1000```)?

Reply:
Hm, let me map out the current behavior and see if it makes sense:
100 kg -> 100 kg
998 kg -> 998 kg
9998 kg -> 9998 kg
9999 kg -> 9.9 k kg

Which does seem wrong.
Yeah I agree that this should be `> 999`.

Also maybe instead of 9.9 k kg, we should use 9.9 `mega gm`?
The standard SI units definitely don't have `kilo kilo`, 10^6 is mega.
On the other hand, `mega gm` looks unwieldy and `Mg` is not very well known.
UX thoughts?

Reply:
I'll change it to `> 999` for now. I can venture that big numbers should be displayed for rhetoric purposes, i.e. as the means to emphasise the amount/impact.